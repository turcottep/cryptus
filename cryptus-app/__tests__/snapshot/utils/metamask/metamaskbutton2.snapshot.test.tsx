import React from "react";
import renderer from "react-test-renderer";
import "@testing-library/jest-dom/extend-expect";

import MetamaskButton2 from "../../../../components/utils/metamask/metamaskbutton2";

describe("<MetamaskButton2 />", () => {
  it("should display properly", async () => {
    const mockProps = {
      session: null,
      setLoading: null,
      isMobile: false,
    };

    const tree = renderer.create(<MetamaskButton2 {...mockProps} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
