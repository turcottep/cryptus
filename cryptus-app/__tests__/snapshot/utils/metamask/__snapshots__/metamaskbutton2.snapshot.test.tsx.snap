// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MetamaskButton2 /> should display properly 1`] = `
<div>
  <div
    onClick={[Function]}
  >
    SIGN IN
  </div>
</div>
`;
